site/src/components/WorkspaceActions/constants.ts

```diff
@@ -1,5 +1,5 @@
+import { WorkspaceStatus } from "api/typesGenerated"
 import { ReactNode } from "react"
-import { WorkspaceStateEnum } from "util/workspace"
 
 // the button types we have
 export enum ButtonTypesEnum {
```

Comment:
I assume these are based on the `WorkspaceStatus`? We could leverage those types here as well.

Reply:
These are similar to `WorkspaceStatus` but not identical.